### PR TITLE
Request handlers

### PR DESCRIPTION
**Pull request by rjgcabrera**
Hey Steve,

Would you be so kind to review my server-side code? Let me know if there are any bugs I need to squash.

Cheers!
-Raymond

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const path = require('path');
+var db = require('./../database/index.js');
```
Comment (kentacohut) [inline server/index.js]: Why not change this to 'const' just to be consistent?

File: database/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+var mysql = require('mysql'); //<-- install
+var Promise = require('bluebird');
+var sqlString = require('sqlString'); //<-- install
+var _ = require('underscore'); //<-- install
+
+
+
+var get = function() { // CHANGE QUERY STRING LATER
```
Comment (kentacohut) [inline database/index.js]: How about changing these to 'const' so that the code is consistent from file to file?

File: database/index.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+var mysql = require('mysql'); //<-- install
+var Promise = require('bluebird');
+var sqlString = require('sqlString'); //<-- install
+var _ = require('underscore'); //<-- install
+
```
Comment (kentacohut) [inline database/index.js]: How about changing the above 'var' tags to 'const' just to be consistent across all the files?

File: database/index.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+
+var connection = mysql.createConnection({
+  host: '127.0.0.1',
+  user: 'root',
+  password: '',
+  database: 'open_source_table'
+});
+
```
Comment (kentacohut) [inline database/index.js]: You could change the above 'var' tags to be 'let' tags to match the other files.

**Comment by rjgcabrera** [comment]
Hey Steve!

I made the requested changes. Let me know what you think.

-Raymond

**Pull request merged**